modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AnalyticsSource.kt

```diff
@@ -15,9 +16,11 @@ enum class AnalyticsSource(val analyticsValue: String) {
     EPISODE_DETAILS("episode_details"),
     MINIPLAYER("miniplayer"),
     PLAYER("player"),
+    WATCH_PLAYER("watch_player"),
     NOTIFICATION("notification"),
     FULL_SCREEN_VIDEO("full_screen_video"),
     UP_NEXT("up_next"),
+    WATCH_UP_NEXT("watch_up_next"),
```

Comment:
Let me know if you think there's a better way to handle "watch" sources than just adding watch to the front of the screen name like I am doing here.

Reply:
Looks fine to me.